tools/bin/build_report.py

```diff
@@ -29,7 +29,7 @@
 SOURCE_DEFINITIONS_YAML = f"{CONNECTOR_DEFINITIONS_DIR}/source_definitions.yaml"
 DESTINATION_DEFINITIONS_YAML = f"{CONNECTOR_DEFINITIONS_DIR}/destination_definitions.yaml"
 CONNECTORS_ROOT_PATH = "./airbyte-integrations/connectors"
-RELEVANT_BASE_MODULES = ["base-normalization", "source-acceptance-test"]
+RELEVANT_BASE_MODULES = ["base-normalization", "bases/source-acceptance-test"]
```

Comment:
All good questions and things that I encountered when I wrote this. So our actions were coded up to explicitly handle the base-normalization input case in `ci_integration_test.sh` as such: https://github.com/airbytehq/airbyte/blob/master/tools/bin/ci_integration_test.sh#L16-L22

That's why it works w/o changes for base-normalization. Alternatively, I can also adjust the test to follow the same pattern. But to me, it kind of feels like more of a hack to write in special logic for the test script to handle each base we are adding instead of using the existing condition to handle things prefixed with `/bases`. And we just need to supply the proper arguments upstream.

And the reason why we don't run into it in for any other bases is because the list of test workflows is generated using `./gradlew integrationTest --dry-run` here in https://github.com/airbytehq/airbyte/blob/master/tools/bin/ci_integration_workflow_launcher.sh#L90 only results in all sources, destinations, base-normalization, and source-acceptance-test.

I'm not super attached to which approach to take, but at the moment the root cause is that we aren't scheduling SAT correctly because of the arguments we're supplying to integrationTest.